research/src/sources/w3.json5

```diff
@@ -106,6 +106,12 @@
       "definition": "Interactive Tabular Data and Layout Containers",
       "url": "https://www.w3.org/TR/wai-aria-practices/#grid"
     },
+    {
+      "name": "Layout Grid",
+      "openUIName": "Flex",
+      "definition": "A grid widget is a container that enables users to navigate the information or interactive elements it contains using directional navigation keys, such as arrow keys, Home, and End. As a generic container widget that offers flexible keyboard navigation, it can serve a wide variety of needs. It can be used for purposes as simple as grouping a collection of checkboxes or navigation links or as complex as creating a full-featured spreadsheet application. While the words row and column are used in the names of WAI-ARIA attributes and by assistive technologies when describing and presenting the logical structure of elements with the grid role, using the grid role on an element does not necessarily imply that its visual presentation is tabular.",
+      "url": "https://www.w3.org/TR/wai-aria-practices/examples/grid/LayoutGrids.html"
+    },
```

Comment:
I think this is represented in lines 105-107, though the layout portion is essentially a sub-type of the "Grid" widget pattern from the W3C. I'm not sure we should duplicate with a more specific example.

Reply:
I don't feel like `Flex` would really represent the W3C's `Grid` widget definition, so perhaps it's best just to remove this connection between them. What do you think?

Reply:
I've gone ahead and removed Layout Grid